code/RealMeAuthenticator.php

```diff
@@ -25,51 +25,26 @@ protected static function on_register()
         $cache = SS_Cache::factory('RealMeAuthenticator');
 
         $cacheKey = 'RegisterCheck';
-        if (true === (bool) $cache->load($cacheKey)) {
+        if ((bool) $cache->load($cacheKey)) {
```

Comment:
Ah, didn't spot that. Howver, if you want false you store `'false'` this would cast to true too